docs/features/wait/host_port.md

```diff
@@ -0,0 +1,31 @@
+# HostPort wait strategy
+
+The host-port wait strategy will check if a port is listening in the container, being able to set the following conditions:
+
+- a port to be listening in the container. The port and protocol to be used, which is represented by a string containing port number and protocol in the format "80/tcp".
+- the first exposed port in the container.
```

Comment:
This means instead of specifying a port, just wait for the first exposed port, correct? Needs a bit more clarification.

The example below is actually useful, I just found this bullet point confusing.

Reply:
Indeed. It would be one or the other, not both. Let me think how to rephrase that.

Reply:
maybe something like:
`- alternatively, to wait for the first exposed port of the container`

Also, container vs image here 😉